folio_data_anonymization/anonymizer.py

```diff
@@ -0,0 +1,24 @@
+import os
+import logging
+from folio_data_anonymization.users import Users
+
+logger = logging.getLogger(__name__)
+
+data: list = ["circulation", "inventory", "organizations", "users"]
+
+
+def anonymize(**kwargs) -> str:
+    logging.basicConfig(filename="anonymizer.log", level=logging.INFO)
+    data = kwargs.get("data", [])
```

Comment:
It'd probably be a bit simpler to just declare a `data` function parameter here, rather than having to extract it from `**kwargs`

```suggestion
def anonymize(data) -> str:
    logging.basicConfig(filename="anonymizer.log", level=logging.INFO)
```

Reply:
I made this suggested change. Thanks!